src/renderer/fonts/font_options.rs

```diff
@@ -1,9 +1,11 @@
-use super::font_loader::FontSelection;
+use super::{font_loader::FontSelection, FontSlant, FontWeight};
 
 const DEFAULT_FONT_SIZE: f32 = 14.0;
 
 #[derive(Clone, Debug)]
 pub struct FontOptions {
+    use_italic_as_oblique: bool,
```

Comment:
Let's place private properties after public. This way - we first see which is most relevant.

Reply:
Seems okay to me, I've just tried to replicate what I see around `Renderer`